als-structure/shared/src/main/scala/org/mulesoft/language/outline/structure/structureImpl/StructureBuilder.scala

```diff
@@ -65,7 +65,7 @@ object KindForResultMatcher {
     ParametersFieldModel.QueryString.value.iri()          -> SymbolKind.Module,
     ParametersFieldModel.UriParameters.value.iri()        -> SymbolKind.Module,
     EndPointModel.Parameters.value.iri()                  -> SymbolKind.Module,
-    OperationModel.Request.value.iri()                    -> SymbolKind.Interface,
+    OperationModel.Request.value.iri()                      -> SymbolKind.Interface,
```

Comment:
please revert & ensure you are using `scalafmt`

Reply:
Ok